pkg/migrate/migrate.go

```diff
@@ -835,10 +836,14 @@ func scaleDownPods(ctx context.Context, w *log.Logger, clientset k8sclient.Inter
 	}
 
 	// wait for all pods to be deleted
+	stopAt := time.Now().Add(timeout)
 	w.Printf("\nWaiting for pods with mounted PVCs to be cleaned up\n")
 	time.Sleep(checkInterval / 16)
 checkPvcPodLoop:
 	for {
+		if timeout != 0 && time.Now().After(stopAt) {
+			return nil, fmt.Errorf("timeout waiting for pods to scale down")
```

Comment:
I would prefer to call scaleuppods before we return this, that way we leave things in an OK state

Reply:
Yeah, if we scale things back up then we increase the risk of having the same issue I was trying to avoid (Longhorn not scaling things down). I am closing this one and resume on Monday.